src/components/search/dataSearchQueryBuilder.js

```diff
@@ -0,0 +1,61 @@
+export const getDataSimpleSearchQuery = (
+    searchTerm,
+    userHomeDir,
+    rowsPerPage,
+    offset
+) => {
+    if (userHomeDir) {
+        return {
+            query: {
+                all: [
+                    {
+                        type: "label",
+                        args: {
+                            exact: false,
+                            negated: false,
```

Comment:
I believe it is being by search service. Here is the results for comparison
- without `negated`
![image](https://user-images.githubusercontent.com/1250790/90139316-32537180-dd46-11ea-8c27-9bbc6fa2c906.png)

- with negated.
![image](https://user-images.githubusercontent.com/1250790/90139397-4f884000-dd46-11ea-8705-0f1a21fd2171.png)

Reply:
I'm not sure what's up here. The searches being created by the backend should be exactly the same with or without `negated` -- it appears that maybe some searches don't include the `path` clause in the backend though -- here's the queries from your user, today:

```
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"tes\"]","service":"search","time":"2020-08-13T06:19:46-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"test\"]","service":"search","time":"2020-08-13T06:19:46-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"tes\",path=\"/iplant/home/sriram/\"]","service":"search","time":"2020-08-13T06:20:44-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"test\",path=\"/iplant/home/sriram/\"]","service":"search","time":"2020-08-13T06:20:45-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"test\",path=\"/iplant/home/sriramNaN\"]","service":"search","time":"2020-08-13T06:30:56-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"tes\",path=\"/iplant/home/sriramNaN\"]","service":"search","time":"2020-08-13T06:30:56-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"tes\",path=\"/iplant/home/sriramNaN\"]","service":"search","time":"2020-08-13T06:32:10-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"tes\",path=\"/iplant/home/sriramNaN\"]","service":"search","time":"2020-08-13T06:32:32-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"test\",path=\"/iplant/home/sriramNaN\"]","service":"search","time":"2020-08-13T06:32:32-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[None:[path=\"/iplant/home/shared\"],label~\"test\"]","service":"search","time":"2020-08-13T06:33:36-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"test\",path=\"/iplant/home/sriramNaN\"]","service":"search","time":"2020-08-13T06:37:43-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"tes\",path=\"/iplant/home/sriramNaN\"]","service":"search","time":"2020-08-13T06:37:43-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"tes\",path=\"/iplant/home/sriramNaN\"]","service":"search","time":"2020-08-13T06:40:53-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"test\",path=\"/iplant/home/sriramNaN\"]","service":"search","time":"2020-08-13T06:40:54-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"tes\",path=\"/iplant/home/sriram/\"]","service":"search","time":"2020-08-13T06:42:48-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"test\",path=\"/iplant/home/sriram/\"]","service":"search","time":"2020-08-13T06:42:48-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"tes\",path=\"/iplant/home/sriram/\"]","service":"search","time":"2020-08-13T06:47:12-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"test\",path=\"/iplant/home/sriram/\"]","service":"search","time":"2020-08-13T06:47:13-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"tes\",path=\"/iplant/home/sriram/\"]","service":"search","time":"2020-08-13T06:58:08-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"test\",path=\"/iplant/home/sriram/\"]","service":"search","time":"2020-08-13T06:58:09-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"tes\",path=\"/iplant/home/sriram/\"]","service":"search","time":"2020-08-13T06:58:14-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"test\",path=\"/iplant/home/sriram/\"]","service":"search","time":"2020-08-13T06:58:14-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"tes\",path=\"/iplant/home/sriram/\"]","service":"search","time":"2020-08-13T06:58:27-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"test\",path=\"/iplant/home/sriram/\"]","service":"search","time":"2020-08-13T06:58:27-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"tes\",path=\"/iplant/home/sriram/\"]","service":"search","time":"2020-08-13T10:39:03-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"test\",path=\"/iplant/home/sriram/\"]","service":"search","time":"2020-08-13T10:39:03-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"test\",path=\"/iplant/home/sriram/\"]","service":"search","time":"2020-08-13T10:39:04-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"test\"]","service":"search","time":"2020-08-13T10:45:02-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"test\",path=\"/iplant/home/sriram/\"]","service":"search","time":"2020-08-13T10:45:04-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"test\"]","service":"search","time":"2020-08-13T10:46:05-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"test\",path=\"/iplant/home/sriram/\"]","service":"search","time":"2020-08-13T10:46:09-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"test\",path=\"/iplant/home/sriram/\"]","service":"search","time":"2020-08-13T10:47:40-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"test\",path=\"/iplant/home/sriram/\"]","service":"search","time":"2020-08-13T10:47:49-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"test\",path=\"/iplant/home/sriram/\"]","service":"search","time":"2020-08-13T10:47:55-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"test\"]","service":"search","time":"2020-08-13T11:05:57-07:00"}
{"art-id":"search","group":"org.cyverse","level":"info","msg":"Got a request from user sriram: All:[label~\"test\",path=\"/iplant/home/sriram/\"]","service":"search","time":"2020-08-13T11:06:05-07:00"}
```

Reply:
Ahh I see what is the confusion..I miss understood that you were talking about the whole `path` clause for some reason. I see you are just talking about the `negated` key ? Is that right ?

Reply:
Yeah, that's correct. Just the `negated` key in each clause should be unnecessary, since that key isn't used -- the rest of the clauses should stick around.

Reply:
Makes sense. Sorry about the confusion.